examples/addon.node/__test__/whisper.spec.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path');
+const { whisper } = require(path.join(__dirname, '../../../build/Release/whisper-addon'));
+
+const whisperParamsMock = {
+    language: 'en',
+    model: path.join(__dirname, '../../../models/ggml-base.en.bin'),
+    fname_inp: path.join(__dirname, './samples_jfk.wav'),
```

Comment:
Maybe better to reuse the `jfk.wav` from the `samples` folder, instead of having a copy here:

```suggestion
    fname_inp: path.join(__dirname, '../../../samples/jfk.wav'),
```